ListManagement/processNewMembers.py

```diff
@@ -195,59 +236,73 @@ def setupEmail() -> EmailAPI.EmailAccount:
     emailAccount = EmailAPI.EmailAccount(mailUsername, mailPassword)
     return emailAccount
 
+
 def dowloadMembershipListFromEmail(emailAccount: EmailAPI.EmailAccount) -> str:
     if os.path.exists(Constants.DOWNLOAD_ZIP_PATH):
-            logging.info("Found old downloaded zip file, deleting")
-            os.remove(Constants.DOWNLOAD_ZIP_PATH)
-            
-    emailAccount.downloadZipAttachmentFromMostRecentUnreadEmail(Constants.MEMBERSHIP_EMAIL,
-                                                                Constants.EXPECTED_EMAIL_SUBJECT,  
-                                                                Constants.DOWNLOAD_ZIP_PATH,
-                                                                datetime.datetime.now() - datetime.timedelta(days=10), 
-                                                                expectedFileName=Constants.EXPECTED_LIST_ATTACHMENT_NAME)
-
-    
+        logging.info("Found old downloaded zip file, deleting")
+        os.remove(Constants.DOWNLOAD_ZIP_PATH)
+
+    emailAccount.downloadZipAttachmentFromMostRecentUnreadEmail(
+        Constants.NOTIFICATION_EMAILS["MEMBERSHIP"],
+        Constants.EXPECTED_EMAIL_SUBJECT,
+        Constants.DOWNLOAD_ZIP_PATH,
+        datetime.datetime.now() - datetime.timedelta(days=10),
+        expectedFileName=Constants.EXPECTED_LIST_ATTACHMENT_NAME,
+    )
+
     if os.path.exists(Constants.DOWNLOAD_LIST_PATH):
         logging.info("Found old downloaded list, deleting")
         os.remove(Constants.DOWNLOAD_LIST_PATH)
 
-    logging.info(f"Unzipping downloaded list to {Constants.DOWNLOAD_LIST_PATH}")
+    logging.info("Unzipping downloaded list to %s", Constants.DOWNLOAD_LIST_PATH)
```

Comment:
Just curious here as I'm not too aware of the semantic difference between f-strings and passing in arguments. Is there a semantic difference or is this a style/linter change?

Reply:
https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation

By passing the strings as args, the interpolation only occurs when the log message is posted. It saves processing time when not running with info messages enabled.

It's also better for log compilers but I don't know how relevant that is here.